eclim-java.el

```diff
@@ -544,8 +544,27 @@ method."
     (compile (concat eclim-executable " -command java -p "  eclim--project-name
                      " -c " (eclim-package-and-class)))))
 
+(defun eclim--java-junit-file (project file offset encoding)
+     (concat eclim-executable
+             " -command java_junit -p " project
+             " -f " file
+             " -o " (number-to-string offset)
+             " -e " encoding))
+
+(defun eclim--java-junit-project (project endcoding)
+     (concat eclim-executable
+             " -command java_junit -p " project
+             " -e " encoding))
+
+(defun eclim--buffer-contains-substring (string)
```

Comment:
Is there function for that somewhere?  Maybe there's way to ask `eclim` whether some file is test files or not. If there is then I would not need this function